extensions/amp-access-laterpay/0.1/laterpay-impl.js

```diff
@@ -437,7 +439,7 @@ export class LaterpayVendor {
     a.href = href;
     a.textContent = this.i18n_.alreadyPurchasedLink;
     this.alreadyPurchasedListener_ = listen(a, 'click', ev => {
-      this.handlePurchase_(ev, href);
+      this.handlePurchase_(ev, href, 'alreadyPurchased');
```

Comment:
Not for the case of the radio buttons, because the `event.target` for `handlePurchase_` is the buy button, not the radio button itself.  The radio options have 2 steps (select option, then buy), where as this one is an immediate step.